client/src/client.rs

```diff
@@ -817,6 +840,18 @@ impl Client {
         events_api::EventIterator::new(self.events_handler(event_filter)?)
     }
 
+    /// Connects through `WebSocket` to async listen for `Iroha` pipeline and data events.
+    ///
+    /// # Errors
+    /// Fails if subscribing to websocket fails
```

Comment:
```suggestion
    /// - Forwards from [`Self::events_handler`]
```

Reply:
When you do that, the person interested in knowing why it could fail, can just look at the implementation and see the full list of error conditions.